pkg/render/manifests.go

```diff
@@ -12,14 +12,14 @@ import (
 )
 
 // RenderClusterManifests renders manifests for a hosted control plane cluster
-func RenderClusterManifests(params *api.ClusterParams, pullSecretFile, outputDir string, externalOauth, includeRegistry bool) error {
+func RenderClusterManifests(params *api.ClusterParams, pullSecretFile, outputDir string, externalOauth, includeRegistry, konnectivityEnabled bool) error {
```

Comment:
This data is already passed in via params. There is no need to add this parameter.